buildSrc/src/testKit/symbolic-link-preserving-tar/build.gradle

```diff
@@ -0,0 +1,35 @@
+import org.elasticsearch.gradle.tar.SymbolicLinkPreservingTar
+
+plugins {
+    id 'base'
+    id 'distribution'
+    id 'elasticsearch.symbolic-link-preserving-tar'
+}
+
+final String source = Objects.requireNonNull(System.getProperty('tests.symbolic_link_preserving_tar_source'))
+
+task buildBZip2Tar(type: SymbolicLinkPreservingTar) {
+    archiveExtension = 'tar.bz2'
```

Comment:
FWIW, another benefit of having this task extend `Tar` is that it contains logic to automatically set the extension based on the compression.